include/cutlass/epilogue/collective/default_epilogue.hpp

```diff
@@ -147,6 +147,41 @@ class DefaultEpilogue {
     return epilogue_op.is_source_needed();
   }
 
+#ifdef EPILOGUE_RELU
```

Comment:
we should not have macros affecting the library. Any arguments should be passed as template arguments.